LiteFarmOrg/LiteFarm
# Lf 2454 : Sensor readings sync fix with ensemble request body

To test:

1. Unclaim sensors using bulk.py. [please connect with me]
2. Add sensors using a CSV file.
3. check if the webhook is able to send sensor readings local database.
